### PR TITLE
[TOPI] Fix x86 conv2d template when tuning with unpacked layout

### PR DESCRIPTION
**Pull request by merrymercy**
Fix tuning x86 conv2d and conv2d_transpose with unpacked layout
- conv2d
Without this fix, I got
```
UnboundLocalError("local variable 'oc_bn' referenced before assignment")
```
- conv2d_transpose
Wihtout this fix, I got
```
IndexError: list index out of range
```

### CODE REVIEW

**Comment by merrymercy** [comment]
cc @kevinthesun

File: topi/python/topi/x86/conv2d_avx_1x1.py [hunk 1]
```diff
@@ -89,6 +89,8 @@ def _schedule_conv_NCHWc(s, cfg, data_vec, kernel_vec, conv_out, last):
             s[kernel_vec].vectorize(oc_block)
         parallel_axis = s[kernel_vec].fuse(oc_chunk, oh)
         s[kernel_vec].parallel(parallel_axis)
+    else:
+        oc_bn = cfg['tile_oc'].size[-1]
```
Comment (kevinthesun) [inline topi/python/topi/x86/conv2d_avx_1x1.py:93]: Move this to top level?

File: topi/python/topi/x86/conv2d_avx_common.py [hunk 2]
```diff
@@ -111,6 +111,8 @@ def _schedule_conv_NCHWc(s, cfg, data_vec, kernel_vec, conv_out, last):
             s[kernel_vec].vectorize(oc_block)
         parallel_axis = s[kernel_vec].fuse(oc_chunk, oh)
         s[kernel_vec].parallel(parallel_axis)
+    else:
+        oc_bn = cfg['tile_oc'].size[-1]
```
Comment (kevinthesun) [inline topi/python/topi/x86/conv2d_avx_common.py:115]: Same.

**Comment by merrymercy** [comment]
@kevinthesun  comments are addressed

**Comment by tqchen** [comment]
@kevinthesun please follow up

**Comment by kevinthesun** [comment]
Thanks @merrymercy  @junrushao1994 @tqchen

**Pull request merged**